src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.scss

```diff
@@ -1,16 +1,56 @@
+@import "../../../../styles/states.scss";
+@import "../../../../styles/global-css-settings.scss";
+
 .monday-style-color-item-wrapper {
+  --border-radius-multiplier: 1;
+
+  &.circle {
+    --border-radius-multiplier: 100;
```

Comment:
Is `100` here just to make sure the corder radius is a perfect circle? If so, and it's not critical, we might as well go with a ridiculously big number, just to make sure it doesn't break in bigger sizes.

Reply:
sure, I'll change it to 99999 (I think it will also be clearer for devs to understand its an arbitrary number)